app/page.tsx

```diff
@@ -0,0 +1,8 @@
+import Image from 'next/image'
+import PlayListPage from './playlist/page'
+
+export default function Home() {
+  return (
+    <PlayListPage />
```

Comment:
Page.tsx is used to define route, Are we  importing the component from another page? 
Looks like there are 2 routes with same content. Can you have a look at this.